restapi/src/main/java/org/cristalise/restapi/ItemRoot.java

```diff
@@ -558,10 +558,16 @@ public Response getJobFormTemplate(
 
                 if (thisJob == null)
                     throw ItemUtils.createWebAppException("Job not found for actPath:"+actPath+" transition:"+transition, Response.Status.NOT_FOUND);
+                
+                CastorHashMap inputs = (CastorHashMap) thisJob.getActProps().clone();
+
+                inputs.put(Script.PARAMETER_AGENT, agent);
+                inputs.put(Script.PARAMETER_ITEM, item);
+                inputs.put(Script.PARAMETER_JOB, thisJob);
 
                 // set outcome if required
                 if (thisJob.hasOutcome()) {
-                    return Response.ok(new OutcomeBuilder(thisJob.getSchema(), false).generateNgDynamicForms(thisJob.getActProps())).build();
+                    return Response.ok(new OutcomeBuilder(thisJob.getSchema(), false).generateNgDynamicForms(inputs)).build();

```

Comment:
Add `thisJob.getActProps()` into the `inputs` map to keep backwards-compatibility.

Reply:
Ah already done, cool.